datafiles/templates/upload.html.st

```diff
@@ -39,6 +39,15 @@ the <code>Maintainer</code> field as above either to commit to supporting the fo
 yourself or to mark it as unsupported.
 </p>
 
+<h3>Group Accounts</h3>
+
+<p>Occasionally organizations want to have a group / organizational account
+for a package that is maintained by a group of people.  The recommended
+approach for these cases is to only do package uploads from individual
+accounts and use the group account only for managing the maintainer list for
+the package.
```

Comment:
I think it would be useful to have a brief explanation _why_ this is the recommended approach, and why group accounts shouldn't have upload permissions.